docs/manual/spring-security-docs-manual.gradle

```diff
@@ -14,13 +14,24 @@ asciidoctor {
 }
 
 asciidoctorj {
+	def docsTag = snapshotBuild ? 'current' : project.version
 	def ghTag = snapshotBuild ? 'main' : project.version
 	def ghUrl = "https://github.com/spring-projects/spring-security/tree/$ghTag"
+	def ghOldSamplesUrl = "https://github.com/spring-projects/spring-security/tree/5.4.x/samples"
+	def ghSamplesUrl = "https://github.com/spring-projects/spring-security-samples/tree/$ghTag"
+	def securityDocsUrl = "https://docs.spring.io/spring-security/site/docs/$docsTag"
+	def securityApiUrl = "$securityDocsUrl/api/"
+	def securityReferenceUrl = "$securityDocsUrl/reference/html5/"
+	def springFrameworkApiUrl = "https://docs.spring.io/spring-framework/docs/current/javadoc-api/"
```

Comment:
Can you look up the `springVersion` Gradle property and apply it here to the URL? That way the Security docs will point to the corresponding Framework docs.

Reply:
Yeah. I apologize because I forgot to answer your question about computing the Spring Framework version. Will do that!

Reply:
The `springVersion` is not available anymore in the project. We are going to stick with the `5.3.x` version for this PR and file a ticket for considering the property.